Braintree.xcodeproj/project.pbxproj

```diff
@@ -2620,7 +2629,6 @@
 			isa = PBXResourcesBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
-				62DE8FBF2B9656BF00F08F53 /* PrivacyInfo.xcprivacy in Resources */,
```

Comment:
Same question here just want to make sure this removal is expected

Reply:
fixed in ba5b597fa1ece265495740a52081676163abbb5d